app/views/notify_mailer/notify_status_change.html.erb

```diff
@@ -1,3 +1,3 @@
 <%= @description %><br>
 Here's the ticket link<br>
-<%= @ticket_link %>
```

Comment:
Dont make this changes, as we are mailing customized link to ticket, also check for other mails, if we are giving better links, why to give a generalized link.